tests/test_derivatives.py

```diff
@@ -247,3 +255,18 @@ def test_shortcuts(self, expr, expected):
 
         assert isinstance(expr, Differentiable)
         assert expected == str(expr)
+
+    @pytest.mark.parametrize('so', [2, 5, 8])
+    def test_all_shortcuts(self, so):
+        """
+        Test that verify that all fd shortcuts are functional
+        """
+        grid = Grid(shape=(10, 10, 10))
+        f = Function(name='f', grid=grid, space_order=so)
+        g = TimeFunction(name='g', grid=grid, space_order=so)
+
+        for fd in f._fd:
+            assert getattr(f, fd)
```

Comment:
should you not explicitly list what you expect to be able to call, here?

Reply:
Not sure what you mean, only tests that all generated shortcuts are callable